### DISCUSSION TITLE
add support for pt100/1000 sensors

### DISCUSSION THREAD

**Discussion by ant1fr1z** [vitotai/BrewManiacEsp8266#123] [Ideas]
Pt100/1000 sensors are very relevant for the distillation process, as they have a smaller diameter and are suitable for a larger number of equipment. This will expand the number of users of brewmaniac automation. I propose to add support for the specified sensors and make the option to select the type of sensor in the settings.

**Comment by vitotai** [comment]
I've responded in issue #113. Test code was provided. I don't use it, neither do I want to setup the hardware just for testing.
